src/diff/props.js

```diff
@@ -35,7 +35,15 @@ let eventClock = 0;
  * @param {string} namespace Whether or not this DOM node is an SVG node or not
  */
 export function setProperty(dom, name, value, oldValue, namespace) {
-	let useCapture;
+	let useCapture, prefix;
+	if (name[0] == 'a' || name[0] == 'p') {
```

Comment:
Would it be easier/mre performant by adding `&& name[5] == ':'` here? This would allow us to remove `idx` and replace it with 5 by default

Reply:
That it would be